docker-compose.yml

```diff
@@ -28,6 +28,8 @@ services:
       # Make it so we can edit the start script dynamically,
       # for example to install dependencies
       - ./docker-compose/django/start:/start
+      # Load the ethicalads_ext code from the host, so we don't have to rebuild
+      - ${PWD}/${EA_EXT_PATH:-../ethicalads-ext}:/ethicalads-ext
```

Comment:
What happens when this directory isn't present? Does it fail silently or loudly?

Reply:
It seems to just load a blank directory there from what I can tell.